.circleci/config.yml

```diff
@@ -21,8 +21,50 @@ jobs:
           working_directory: javascript
           command: docker build -t creditkudos-development.com/auth-demo .
 
+  go_build:
```

Comment:
Can you rename this to reflect that it's building a docker image not a go binary?